src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -91937,4 +91937,62 @@ CREATE VIEW VIEW_KNOWN_TRANSPORTER_CONTAINERS AS SELECT
         t.CONTAINER_NAME AS ContainerName,
         t.MEMBRANE AS MembraneLocation
 FROM tab_known_transporters_containers t;
+CREATE VIEW [ContainerParameters_Species]

+AS

+SELECT DISTINCT 

+                [cpc].[container_id], 

+                [cpc].[container_type], 

+                [cpc].[container_name], 

+                [cpc].[parameter_name], 

+                [species]

+FROM   [tab_container_parameter_curves] AS [cpc],

+       [tab_container_parameters] AS [cp]

+WHERE  [cpc].[container_id] = [cp].[container_id]

+         AND [cpc].[parameter_name] = [cp].[parameter_name]

+         AND [building_block_type] = "INDIVIDUAL"

+UNION

+SELECT DISTINCT 

+                [cpv].[container_id], 

+                [cpv].[container_type], 

+                [cpv].[container_name], 

+                [cpv].[parameter_name], 

+                [species]

+FROM   [tab_container_parameter_values] AS [cpv],

+       [tab_container_parameters] AS [cp]

+WHERE  [cpv].[container_id] = [cp].[container_id]

+         AND [cpv].[parameter_name] = [cp].[parameter_name]

+         AND [building_block_type] = "INDIVIDUAL"

+UNION

+SELECT DISTINCT 

+                [cpr].[container_id], 

+                [cpr].[container_type], 

+                [cpr].[container_name], 

+                [cpr].[parameter_name], 

+                [species]

+FROM   [tab_container_parameter_rates] AS [cpr],

+       [tab_species_calculation_methods] AS [scm],

+       [tab_container_parameters] AS [cp]

+WHERE  [cpr].[calculation_method] = [scm].[calculation_method]

+         AND [cpr].[container_id] = [cp].[container_id]

+         AND [cpr].[parameter_name] = [cp].[parameter_name]

+         AND [building_block_type] = "INDIVIDUAL";
+CREATE VIEW [VIEW_CONTAINER_PARAMETER_NOT_FOR_ALL_SPECIES]

+AS

+SELECT DISTINCT 

+                [ContainerParameters_Species].[container_id] AS [ContainerId], 

+                [ContainerParameters_Species].[container_type] AS [ContainerType], 

+                [ContainerParameters_Species].[container_name] AS [ContainerName], 

+                [ContainerParameters_Species].[parameter_name] AS [ParameterName], 

+                COUNT ([ContainerParameters_Species].[species]) AS [SpeciesCount]

+FROM   [ContainerParameters_Species]

+GROUP  BY

+          [container_id], 

+          [container_type], 

+          [container_name], 

+          [parameter_name]

+HAVING COUNT ([species]) < (SELECT COUNT ([species])

+       FROM   [tab_species])

+ORDER  BY

+          [container_id],

+          [parameter_name];

```

Comment:
This is the view used by PK-Sim. It retrieves the table
|ContainerId|ContainerType|ContainerName|ParameterName|SpeciesCount|
|--|--|--|--|--|

Species count tells in how many species the parameter is defined (currently this info is not used by PK-Sim)